app/admin/family.rb

```diff
@@ -129,7 +129,7 @@
                 mtl_spouse += amount
               when /MPD.*/.match?(conference.name)
                 mpd += amount
-              when conference.name == 'Legacy Track (Staff 60 years of age and older)'
+              when /Legacy.*/.match?(conference.name)
```

Comment:
anchor that to beginning of the string?

Reply:
Or, if you merely must match "Legacy" anywhere, the pattern could be plain `/Legacy/`, although in this case regex is overkill and you could just use `.include?`